.github/workflows/docspublish.yml

```diff
@@ -6,13 +6,48 @@ on:
       - master
 
 jobs:
+  update-cli-screenshots:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          token: ${{ secrets.PERSONAL_ACCESS_TOKEN }}
+          fetch-depth: 0
+
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: "3.x"
+
+      - name: Install dependencies
+        run: |
+          python -m pip install -U pip poetry
+          poetry --version
+          poetry install
+
+      - name: Update CLI screenshots
+        run: |
+          poetry run python scripts/gen_cli_help_screenshots.py
+
+      - name: Commit and push updated CLI screenshots
+        run: |
+          git config --global user.name "github-actions[bot]"
+          git config --global user.email "github-actions[bot]@users.noreply.github.com"
+          git add docs/images/cli_help
+          git commit -m "docs(cli/screenshots) update CLI screenshots" -m "[skip ci]"
+          git push
+
   publish-documentation:
     runs-on: ubuntu-latest
+    needs: update-cli-screenshots
     steps:
       - uses: actions/checkout@v4
         with:
           token: "${{ secrets.PERSONAL_ACCESS_TOKEN }}"
           fetch-depth: 0
+      - name: Pull latest changes
```

Comment:
nitpick: we probably would want to unify the style. (i.e., whether to add a blank line between steps or not)

Reply:
I like to add blank line between steps but since `docspublish.yml` wasnt maded by me I dont like to change without permission. So we follow the blank line after steps?

Reply:
Just checked, looks like our convention is not to have blank lines so let's remove it.

Reply:
Also, it's perfectly fine to change others' code 🙂  If you have doubt, you can leave a comment or make those changes in a single commit in case we decide to revert it

Reply:
Got it! Thanks!

Reply:
I removed the blank lines an ammend it. Waiting for other maintainers to review 😄

Reply:
It's not functionality-related. I'll just merge this one once CI passes

Reply:
@marcosdotme I think we probably need to add *.svg to pre-commit `trim trailing whitespace` exceptions

Reply:
@Lee-W CI passes first time, but when I remove the blank lines and ammend it, it fails 🙃. Trying to figure out why...